server/routes/router.js

```diff
@@ -13,55 +14,88 @@ router.get('/', (req, res) => {
 
 // Auth Endpoints
 router
-  .post('/auth/signup', userValidation.validateSignup, userController.userSignup)
+  .post(
+    '/auth/signup',
+    authorization.createUserType,
```

Comment:
instead of you having to repeat yourself, you can destructure `createUserType`, `adminStaffOnly`, `generalToken` etc from `authorization`